dev.mk.in

```diff
@@ -201,9 +201,11 @@ shellcheck: test/suites/*.bash
 format:
 	$(CLANG_FORMAT) -i $(non_third_party_headers) $(non_third_party_sources) $(test_sources)
 
-# pip install compiledb
 compile_commands.json:
 	$(COMPILEDB) -n $(MAKE) all unittest
+	# Remove redundant './' in front of include paths as this causes clang-tidy
+	# to believe it's a different path (fixes are applied twice).
+	sed -i 's/\-I.\//-I/' compile_commands.json
```

Comment:
In principle the dot needs to be escaped but the dash doesn't, so:

```suggestion
	sed -i 's/-I\.\//-I/' compile_commands.json
```

Or even something like

```suggestion
	sed -i 's!-I\./!-I!' compile_commands.json
```

to avoid a tiny bit of leaning toothpicks syndrome. :slightly_smiling_face:

Reply:
Looks like I'll be learning how to write tests for shell scripts in the next days :-)
(and more about sed than I ever wanted to know)